e2etest/src/test/java/fk/prof/recorder/utils/Util.java

```diff
@@ -76,7 +76,7 @@
 
         return cast(temp.get(fields[i]));
     }
-
+    
```

Comment:
Maybe we use IntelliJ's "Reformat Code" option. It has a way to reformat only stuff that has changed since last commit.